zenhappens/src/services.html

```diff
@@ -0,0 +1,371 @@
+            <td>Lips</td>
+            <td class="table__price">$8</td> 
+        </tr>
+        <tr>
+            <td class="table__add-detail">with lower & upper</td>
+            <td class="table__price">$14</td> 
+        </tr>
+        <tr>
+            <td>Chin</td>
+            <td class="table__price">$10</td> 
+        </tr>
+        <tr>
+            <td>Sideburn</td>
+            <td class="table__price">$12</td> 
+        </tr>
+        <tr>
+            <td>Full Face</td>
+            <td class="table__price">$34</td> 
+        </tr>
+        </table>
+
+        <table class="table-style">
+           <tr>
+            <td>Bikini</td>
+            <td class="table__price">$35</td>
+        </tr>
+        <tr>
+            <td>Extended Bikini</td>
+            <td class="table__price">$40</td> 
+        </tr>
+        <tr>
+            <td>Brazilian</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        <tr>
+            <td>Stomach strip</td>
+            <td class="table__price">$8</td> 
+        </tr>
+        <tr>
+            <td>Inner thigh</td>
+            <td class="table__price">$16</td> 
+        </tr>
+        </table>
+        </div>
+
+
+         <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
+            <td>Back</td>
+            <td class="table__price">$35</td>
+        </tr>
+        <tr>
+            <td>Lower/Upper Back</td>
+            <td class="table__price">$40</td> 
+        </tr>
+        <tr>
+            <td class="table__add-detail">Underarm</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        <tr>
+            <td>Shoulder</td>
+            <td class="table__price">$8</td> 
+        </tr>
+        <tr>
+            <td>Chest</td>
+            <td class="table__price">$16</td> 
+        </tr>
+        <tr>
+            <td>Stomach</td>
+            <td class="table__price">$28</td> 
+        </tr>
+         <tr>
+            <td>Chest/Stomach</td>
+            <td class="table__price">$58</td> 
+        </tr>
+        </table>
+
+        <table class="table-style">
+           <tr>
+            <td>Leg (Half)</td>
+            <td class="table__price">$40</td>
+        </tr>
+        <tr>
+            <td>Leg (Full)</td>
+            <td class="table__price">$68</td> 
+        </tr>
+        <tr>
+            <td>Arm (Half)</td>
+            <td class="table__price">$36</td> 
+        </tr>
+        <tr>
+            <td>Arm (Full)</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        </table>
+        </div>
+        </section>
+        </section>
+
+    
+    <section class="container">
+        <h2 class="price-list__title">Dry Cupping</h2>
+        <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
+            <td rowspan="2" class="u-sansserif">
+            <input type="checkbox" class="read-more-state" id="cupping"/>
```

Comment:
Since the heading is already outside of the table, is there any reason for this text to also be in the table? I'd recommend using the table only for the pricing.